README.md

```diff
@@ -144,6 +144,56 @@ In this configuration, the image is:
 </configuration>
 ```
 
+### Authentication Methods
+
+Pushing/pulling from private registries require authorization credentials. These can [retrieved using Docker credential helpers](#using-docker-credential-helpers) or [defined in your Maven settings](#using-maven-settings).
+
+#### Using Docker Credential Helpers
+
+Docker credential helpers are CLI tools that handle authentication with various registries.
+
+Some common credential helpers include:
+
+* Google Container Registry: [`docker-credential-gcr`](https://cloud.google.com/container-registry/docs/advanced-authentication#docker_credential_helper)
+* AWS Elastic Container Registry: [`docker-credential-ecr-login`](https://github.com/awslabs/amazon-ecr-credential-helper)
+* Docker Hub Registry: [`docker-credential-*`](https://github.com/docker/docker-credential-helpers)
+<!--* Azure Container Registry: [`docker-credential-acr-*`](https://github.com/Azure/acr-docker-credential-helper)
+-->
+
+Configure credential helpers to use by specifying them in the `credHelpers` configuration.
+
+*Example configuration:* 
+```xml
+<configuration>
+  ...
+  <credHelpers>
+    <credHelper>osxkeychain</credHelper>
+  </credHelpers>
+  ...
+</configuration>
+```
+
+#### Using Maven Settings
+
+Registry credentials can be added to your [Maven settings](https://maven.apache.org/settings.html). These credentials will be used if credentials could not be found in any specified Docker credential helpers. 
+
+*Example `settings.xml`:*
+```xml
+<settings>
+  ...
+  <servers>
+    ...
+    <server>
+      <id>MY_REGISTRY</id>
+      <username>MY_USERNAME</username>
+      <username>MY_PASSWORD</username>
```

Comment:
`<username>` -> `<password>`
`MY_PASSWORD` -> `MY_AUTH_TOKEN`

Reply:
`MY_SECRET`? `Secret` is used in Docker credentials store.